controllers/task.js

```diff
@@ -417,6 +418,126 @@ const getGroupByTasks = async (req, res, next) => {
 }
 exports.getGroupByTasks = getGroupByTasks;
 
+const exportDataToExcel = async (req, res, next) => {
+	let data = req.data;
+
+	let allowedTaskGroup = process.env.ALLOWED_GROUP_BY.split(',')
+
+	if (!allowedTaskGroup.includes(data.groupBy)) {
+		return res.status(400).send(sendResponse(400, `${data.groupBy} Group By Not Supported`, 'exportDataToExcel', null, req.data.signature))
+	} else {
+		data.aggregateGroupBy = `$${data.groupBy}`
+	}
+
+	data.aggregateGroupBy = CONSTANTS.GROUP_BY[data.groupBy]
+
+	let taskRes = await createPayloadAndGetGroupByTask(data)
+	let taskData = taskRes.data
+	// data.filteredData taskRes.data;
+	console.log(taskData.tasks)
+	if (taskRes.error) {
```

Comment:
just suggestion for code writing no need to  add space after if and curly bracket

Reply:
Corrected it sir! Thank you for the suggestion